recipe/meta.yaml

```
@@ -29,25 +32,25 @@ requirements:
     - probableparsing
 
 test:
+  source_files:
+    - other/tests
   imports:
     - usaddress
   requires:
     - pip
     - pytest
-  source_files:
-    - tests
   commands:
     - pip check
     # ignore test_labeling.py because it depends on test files and it's about performance,
     # and test_tagging.py because one must train the model (parserator train --trainfile FILES) 
     # to create the usaddr.crfsuite file before one can use the parse and tag methods
-    - pytest tests --ignore tests/test_labeling.py --ignore=tests/test_tagging.py
+    - pytest other/tests --ignore=other/tests/test_labeling.py --ignore=other/tests/test_tagging.py
```

```suggestion
    - pytest other/tests --ignore=other/tests/test_labeling.py
```
Now that we have `usaddr.crfsuite`?

---

correct, will give it a try.